lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -334,25 +334,41 @@ def self.canonicalize_and_validate(hash)
       hash
     end
 
+    V2_SUPPORTED_INTERFACE_MAP =
```

Comment:
Strong prefer on using this instead of mapping in the case.

Reply:
Yeah, even better... we already has this defined in `V2` ... updated to use that instead.